internal/graphql/api.go

```diff
@@ -42,7 +42,7 @@ func AddRoutes(router gin.IRouter, bindplane server.BindPlane) {
 	router.POST("/graphql", gin.WrapH(srv))
 	router.GET("/graphql", gin.WrapH(srv))
 
-	bindplane.Logger().Info(fmt.Sprintf("connect to %s/v1/playground for GraphQL playground", bindplane.Config().BindPlaneURL()))
+	bindplane.Logger().Info(fmt.Sprintf("connect to %s/v1/playground for GraphQL playground", bindplane.BindPlaneURL()))
```

Comment:
I know this already existed, but could we update this?
```suggestion
	bindplane.Logger().Sugar().Infof("connect to %s/v1/playground for GraphQL playground", bindplane.BindPlaneURL())
```